requirements.txt

```diff
@@ -1,13 +1,6 @@
-# This pulls in all the libraries needed to run Selenium tests
-# on Mozilla WebQA projects
-
-PyYAML==3.10
-execnet==1.0.9
-certifi==0.0.8
-chardet==1.0.1
-py==1.4.26
-pytest==2.7.0
+browserid
```

Comment:
I've been commenting on other patches that this should be `bidpom`, but it might not be simply a case of changing the package name. For this reason I think updating this package should be dealt with in a separate patch.